src/PullRequestFindingsTab/PullRequestFindingsTab.ts

```diff
@@ -111,8 +112,8 @@ async function openFindingsKioskAsDialog(pullRequestId: number, teamscaleUrl: st
             protocol: teamscaleProtocolAndHost[0],
             teamscaleServer: teamscaleProtocolAndHost[1],
             teamscaleProject: teamscaleProject,
```

Comment:
Do we need to encode the project id (or alias) as well?

Reply:
I would expect the project id to be alphanumeric without special characters and umlauts